packages/my_at_localization/src/my_at_pose_node.py

```diff
@@ -149,15 +159,15 @@ def _broadcast_detected_tag(self, image_msg, tag_id, tag):
         self._broadcast_tf(
             parent_frame_id=tag_frame_id,
             child_frame_id=tag_cam_frame_id,
-            euler_angles=(-90 * math.pi / 180, 0, 90 * math.pi / 180)
+            euler_angles=(-90 * math.pi / 180, 0, -90 * math.pi / 180)
```

Comment:
These are now the same value twice. Is that what you intended?

Reply:
Yes.